pkg/cqrs/sqlitecqrs/cqrs.go

```diff
@@ -326,6 +375,11 @@ func (w wrapper) GetFunctionRunsFromEvents(ctx context.Context, eventIDs []ulid.
 	}, []*cqrs.FunctionRun{})
 }
 
+func (w wrapper) GetFunctionRun(ctx context.Context, workspaceID uuid.UUID, id ulid.ULID) (*cqrs.FunctionRun, error) {
```

Comment:
wasn't the wrapper initially for duckdb? do we still need it?

Reply:
We still need to map CQRS -> sqlite, right?

Reply:
yeah, didn't expect sqlite will be the same as well since it's a lot more like general sql.